specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2022-11-01-preview/agfood.json

```diff
@@ -13018,6 +13045,168 @@
         }
       }
     },
+    "/scenes/stac-collections/{collectionId}:search": {
+      "post": {
+        "tags": [
+          "Scenes"
+        ],
+        "description": "Search for STAC features by collection id, bbox, intersecting geometry, start and end datetime.",
+        "operationId": "Scenes_SearchFeatures",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "collectionId",
+            "description": "Collection Id to be searched.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "Sentinel_2_L2A",
+              "Sentinel_2_L1C"
+            ],
+            "x-ms-enum": {
+              "name": "collectionId",
+              "modelAsString": true
+            }
```

Comment:
One more issue that you can resolve in this PR or in future api-version (before you request SDK, this won't affect REST API):

Unify the name of the extensible enum.
Here it is `"name": "Source"`
https://github.com/Azure/azure-rest-api-specs/blob/e38daec67d57ef9c4804b1e3055753407e45fa71/specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2022-11-01-preview/agfood.json#L18473-L18481

Though we call it extensible enum, it more akin to "string with known values" that has a "name" in SDK. And here we'd like same extensible enum having same name.

Reply:
Sure. Will update in future versions. Although the values are same now, they can diverge in future. Hence kept different names for future flexibility.